[SOAR15334]- Updated email validator to show a warning if the example did not match (#171)
rapid7/icon-integrations-validators: README.md

## Changelog

> * 2.47.7 - Fixed ConfidentialValidator to show a warning when the email example does not match

Reviewer: Personal preference, the message dosent have to be the same but the start of it: `ConfidentialValidator - <desc>` is a nice format.
```suggestion
* 2.47.7 - `ConfidentialValidator` - Changed email violation to a warning 
```

Author: Sounds good!
